src/ape/api/accounts.py

```diff
@@ -59,7 +59,7 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
         Sign a message.
 
         Args:
-          msg (:class:`~ape.types.signatures.SignableMessage`): The message to sign.
+          msg (:class:`~ape.types.signatures.   SignableMessage`): The message to sign.
```

Comment:
Why this have a space?
```suggestion
          msg (:class:`~ape.types.signatures.SignableMessage`): The message to sign.
```

Reply:
i wish there was a fat-finger emoji